src/locale/locales/de/messages.po

```diff
@@ -192,37 +187,33 @@ msgstr ""
 msgid "Appeal Content Warning"
 msgstr ""
 
-#: src/view/com/modals/AppealLabel.tsx:65
-#~ msgid "Appeal Decision"
-#~ msgstr "Berufungsentscheidung"
-
 #: src/view/com/util/moderation/LabelInfo.tsx:52
 msgid "Appeal this decision"
 msgstr "Einspruch gegen diese Entscheidung"
```

Comment:
The verb is missing here. "Einspruch gegen dieses Entscheidung einlegen"
But we could abbreviate it with "Einspruch einlegen" since the context is known